modules/govuk/manifests/apps/content_data_admin.pp

```diff
@@ -62,6 +62,18 @@
 #   Whether to enable the procfile worker
 #   Default: true
 #
+# [*aws_access_key_id*]
+#   The Access Key ID for AWS to access S3 buckets.
+#
+# [*aws_secret_access_key*]
+#   The Secret Access Key for AWS to access S3 buckets.
+#
+# [*aws_region*]
+#   The Region for AWS to access S3 buckets.
+#
+# [*aws_s3_bucket_name*]
+#   The S3 Bucket for AWS to access.
```

Comment:
It might be good if this comment could be a little more specific to the Content Data Admin, e.g. "The S3 Bucket in which to upload CSV files to".